parler/fields.py

```diff
@@ -13,6 +13,10 @@
 
 import django
 from django.forms.forms import pretty_name
+from parler.utils.i18n import get_language
+
+if (1, 8) <= django.VERSION < (2, 0):
+    from compositefk.fields import RawFieldValue, CompositeOneToOneField
```

Comment:
I'm fine with dropping Django 1.7 support, but would love to have Django 2.0 support in.

Reply:
Added support Django 2 to django-composite-foreignkey recently. So can remove the 'if' here.